### ISSUE TITLE
Prepare announcement for Yammer

### ISSUE THREAD

**Issue by andreabrambilla** [equinor/esv-intersection#344]

**Comment by andreabrambilla** [comment]
```
I am happy to announce that version 1.0 of the _esv-intersection_ [TODO: link to repo] component has been released! The esv-intersection component can be used in web application for visualizing subsurface data along a well. It has been developed in collaboration by the VidEx team (GBS IT SI SIS1) [TODO link] and the HeapPurple team (GBS IT SI SIB) [TODO link], and it is currently used in production in the REP and WellX projects [TODO link].
If you would like to explore the available features, try out our storybook [TODO link]
For more details, see the README  [TODO] in the repository
And if you have any question, simply join the #esv-intersection channel on Slack
```

**Comment by andreabrambilla** [comment]
NOTE: point out that the storybook has been moved and that the instance on Radix will be terminated at the end of august

NOTE: the README still links to the esv repo, which is private. We should either make it internal, or remove the reference from the README

**Comment by venikx** [comment]
I noticed esv-intersection has been released to v1.5.0 already, so I assume the issue can be closed?

**Comment by schoening** [comment]
Closing.
